bin/setup

```diff
@@ -1,11 +1,17 @@
 #!/usr/bin/env ruby
 require 'pathname'
 
+def system(*args)
+  puts args.first
+  super
+end
+
 gem_root = Pathname.new(__dir__).join("..")
 
 unless gem_root.join("spec/manageiq").exist?
   puts "== Cloning manageiq sample app =="
   system "git clone https://github.com/ManageIQ/manageiq.git --branch master --depth 1 spec/manageiq"
+  system "find spec/manageid -name '*_spec.rb' -delete"
```

Comment:
This might cause us troubles when calling `git pull` on [bin/update](https://github.com/cfcosta/manageiq-providers-hawkular/blob/eff59769d1babb023ebf41ce86f0cb6f79bf1df4/bin/update#L6-L7).
We maybe (I'm not exactly sure if there could be conflicts if the deleted files were modified) revert back those files (`git reset --hard` maybe?) previous to the `git pull` and redelete the specs (some newer specs might have been added).